COM_dev:
 /dev/arm

